File: src/main/resources/output/IFC4X3_python/Entities/IfcCoordinateOperation/IfcRigidOperation/IfcRigidOperation.py
class IfcRigidOperation(IfcCoordinateOperation):
    def __init__(self):
        self.FirstCoordinate: IfcMeasureValue = None
        self.SecondCoordinate: IfcMeasureValue = None
        self.Height: IfcLengthMeasure = None

    # === WHERE CLAUSES ===
    # SameCoordinateType : (('IFC4X3_DEV_73740fe4.IFCLENGTHMEASURE' IN TYPEOF(FirstCoordinate)) AND ('IFC4X3_DEV_73740fe4.IFCLENGTHMEASURE' IN TYPEOF(SecondCoordinate))) OR (('IFC4X3_DEV_73740fe4.IFCPLANEANGLEMEASURE' IN TYPEOF(FirstCoordinate)) AND ('IFC4X3_DEV_73740fe4.IFCPLANEANGLEMEASURE' IN TYPEOF(SecondCoordinate)))

File: src/main/resources/output/IFC4X3_java/Entities/IfcCoordinateOperation/IfcRigidOperation/IfcRigidOperation.java
public class IfcRigidOperation extends IfcCoordinateOperation {
    public IfcMeasureValue FirstCoordinate;
    public IfcMeasureValue SecondCoordinate;
    public IfcLengthMeasure Height;

    // === WHERE CLAUSES ===
    // SameCoordinateType : (('IFC4X3_DEV_73740fe4.IFCLENGTHMEASURE' IN TYPEOF(FirstCoordinate)) AND ('IFC4X3_DEV_73740fe4.IFCLENGTHMEASURE' IN TYPEOF(SecondCoordinate))) OR (('IFC4X3_DEV_73740fe4.IFCPLANEANGLEMEASURE' IN TYPEOF(FirstCoordinate)) AND ('IFC4X3_DEV_73740fe4.IFCPLANEANGLEMEASURE' IN TYPEOF(SecondCoordinate)))
}
